docs/src/integrations/activity/reference.md

```diff
@@ -2,364 +2,715 @@
-* `integration.health.pagerduty`: Health event sent to PagerDuty.
-* `integration.health.slack`: Health event sent to slack.
-* `integration.webhook`: Webhook triggered.
-* `integration.script`: An activity script has run.
+The `type` value is a combination of the scope and the action itself for that event.
+Its scope can be in one of the following categories:
+
+- [Project](#project-event-value)
+- [Environment](#environment-event-value)
+- [Integration](#integration-event-value)
+
+#### `Project` event value
+
+This value describes the action that happened on a given project, its value can be any from the following table.
+
+| Name | Description |
+|------|-------------|
+| `project.modify.title` | The human-friendly title of the project has been changed. |
+| `project.create` | A project has been created. Although it appears in the activity feed exactly once, it's not sent via a webhook as it always happens before a webhook can be configured. |
+| `project.variable.create` | A new project variable has been created. |
+| `project.variable.delete` | A project variable has been deleted. |
+| `project.variable.update` | A project variable has been modified. |
+
+For an overview of the different actions that can take place, see [the type](#type).
+
+#### `Environment` event value
+
+This value describes the action that happened on a project's environment, its value can be any from the following table.
+
+| Name | Description |
+|------|-------------|
+| `environment.domain.create` | A new domain has been associated with the environment. |
+| `environment.domain.delete` | A domain associated with the environment has been removed. |
+| `environment.domain.update` | A domain associated with the environment has been updated, such as having its SSL certificate modified. |
+| `environment.backup` |  A user triggered a [backup](../../environments/backup.md). |
+| `environment.backup.delete` | A user deleted a [backup](../../environments/backup.md) |
+| `environment.restore` | A user restored a [backup](../../environments/backup.md). |
+| `environment.push` | A user has pushed code to a branch, either existing or new. |
+| `environment.branch` | A new branch has been created via the Console. (A branch created via a push shows up as `environment.push`.) |
+| `environment.activate` | A branch has been "activated", and an environment created for it. |
+| `environment.initialize` | The default branch of the project has just been initialized with its first commit. |
+| `environment.deactivate` | A branch has been "deactivated". The code is still there, but the environment was destroyed. |
+| `environment.synchronize` | An environment has had its data and/or code re-copied from its parent environment. |
+| `environment.merge` | A branch was merged through the Console or Platform.sh API. A basic Git merge doesn't trigger this event. |
+| `environment.redeploy` | An environment was redeployed. |
+| `environment.delete` | A branch was deleted. |
```

Comment:
Two things:
1. It's slightly confusing for an `environment.delete` branch to be describing a `branch`.
2. There are inconsistencies with how `delete` is used in the Console and CLI. Sometimes it deletes an environment and sometimes it just deletes data. It can be confusing, so I'm not sure what's covered by this event.

Reply:
This deletes the environment's branch code through Git.